README.md

```diff
@@ -0,0 +1,4 @@
+Notes:
+
+* The cwebp binary comes from the org.freedesktop.Platform runtime.  If a more specific version is needed, it could be included/built as a module instead.
```

Comment:
The origin of the cwebp binary confused me at first, because it wasn't listed as a dependency/module, so I had to spend some time looking into it.  I've been an OS packager before, so it stood out as odd.  I wanted to save other people the trouble by making a note.  And the second sentence is related - since cwebp isn't explicitly called out as a dependency, this flatpak has no control over it, which means it's making assumptions about the version and capabilities of cwebp.  That could cause unexpected issues in the future if the freedesktop runtime changes, even if someone just rebuilds this flatpak and isn't expecting a real change.  It's not reproducible, in other words, which is different enough from the other dependencies (called out by version and checksum) that I thought it was worth a comment.

Reply:
fair enough